src/frontends/onnx/tests/CMakeLists.txt

```diff
@@ -10,22 +10,23 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND NOT ENABLE_SANITIZER)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fuse-ld=gold")
 endif()
 
-message(STATUS "ONNX frontend tests enabled")
+message(STATUS "ONNX frontend test enabled")
```

Comment:
Why did you rename `tests` to `test`?

Reply:
Plurality derives from the composition of the program, while from higher level perspective it's a single task - to validate this particular functionality. I consider such approach as industry common practice.

Reply:
I propose to have an alignment between OV binaries first of all :)